modules/post-install-edge-aws-extend-machineset.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="post-install-edge-aws-extend-machineset"]
+= Creating a machine set manifest for AWS Local Zones node
+
+After you created subnets in an AWS Local Zones, you can create a machine set manifest.
+
+The installation program sets the following labels for the `edge` machine pools at cluster installtion time:
+
+* `machine.openshift.io/parent-zone-name: <value_of_ParentZoneName>`
+* `machine.openshift.io/zone-group: <value_of_ZoneGroup>`
+* `machine.openshift.io/zone-type: <value_of_ZoneType>`
+
+The following example in the procedure details how you can create a machine set configuraton that matches the `edge` compute pool configuration.
+
+.Procedure
+
+* To preserve those labels when creating the machine set manifest manually, you must gather the AWS API by running the following command:
+
+[source,terminal]
+----
+$ aws ec2 describe-availability-zones --region <value_of_Region> \
+    --query 'AvailabilityZones[].{
+	ZoneName: ZoneName,
+	ParentZoneName: ParentZoneName,
+	GroupName: GroupName,
+	ZoneType: ZoneType}' \
+    --filters Name=zone-name,Values=<value_of_ZoneName> \
+    --all-availability-zones
+----
+
+where:
+
+<value_of_Region>:: Specifies the name of the region for the zone.
+<value_of_ZoneName>:: Specifies the name of the Local Zone.
```

Comment:
My same comment as above, I think these need to be code callouts instead of the current presentation. See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output